src/routes/categories/containers/LanguageFailure.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import PropTypes from 'prop-types'
+import LanguageSelector from 'modules/common/containers/LanguageSelector'
+import setLanguageChangeUrls from 'modules/language/actions/setLanguageChangeUrls'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import withFetcher from 'modules/endpoint/hocs/withFetcher'
+import LanguageModel from 'modules/endpoint/models/LanguageModel'
+import LocationModel from '../../../modules/endpoint/models/LocationModel'
+import Caption from '../../../modules/common/components/Caption'
+
+export class LanguageFailure extends React.PureComponent {
+  static propTypes = {
+    setLanguageChangeUrls: PropTypes.func.isRequired,
+    t: PropTypes.func.isRequired,
+    location: PropTypes.string.isRequired,
+    locations: PropTypes.arrayOf(PropTypes.instanceOf(LocationModel)).isRequired,
+    languages: PropTypes.arrayOf(PropTypes.instanceOf(LanguageModel)).isRequired
+  }
+
+  /**
+   * The function used to map different languages to their CategoriesPage
+   * @param {string} language The language
+   * @returns {string} The url of the CategoriesPage of a different language
+   */
+  mapLanguageToUrl = (language) => `/${this.props.location}/${language}`
+
+  /**
+   * Gets and stores the available languages for the current page
+   */
+  setLanguageChangeUrls () {
+    this.props.setLanguageChangeUrls(this.mapLanguageToUrl, this.props.languages)
+  }
+
+  componentWillMount () {
+    this.setLanguageChangeUrls()
+  }
+
+  getTitle () {
+    return this.props.locations.find(location => location.code === this.props.location).name
+  }
+
+  render () {
+    return <React.Fragment>
+      <Caption title={this.getTitle()} />
+      <p style={{
+        textAlign: 'center',
+        margin: '25px 0'
```

Comment:
do we want to inline styles?

Reply:
I think it doesn't matter here. If it's just two / three lines of css. Especially complex css-code like for the header should be extracted into classes to improve readabilty imo.

Reply:
I think we shouldnt even inline simple ones. We should stay consistent accross all styles. We already have some 2-3 line styles.

Reply:
Agreed. :P